src/@next/components/organisms/CreditCardForm/CreditCardFormContent.tsx

```diff
@@ -29,6 +30,7 @@ export const CreditCardFormContent: React.FC<PropsWithFormik> = ({
   labelsText: { ccCsc: ccCscText, ccExp: ccExpText, ccNumber: ccNumberText },
   handleSubmit,
   handleChange,
+  values,
 }: PropsWithFormik) => {
   const basicInputProps = getInputProps(disabled, handleChange);
```

Comment:
This component will rerender on every input change, it could be beneficial to take advantage of `useCallback` here, to avoid recreating new reference to getInputProps on each render.